code/modules/mob/living/init_signals.dm

```diff
@@ -79,6 +79,8 @@
 	if(buckled && buckled.buckle_lying != NO_BUCKLE_LYING)
 		return // Handled by the buckle.
 	mobility_flags &= ~MOBILITY_STAND
+	if(HAS_TRAIT_FROM(src, TRAIT_IMMOBILIZED, CRYO_TRAIT))
+		return // Don't go horizontal if mob falls asleep from cryo while in cryotube
```

Comment:
Paging @Rohesie for this, this seems like a rather hacky way of preventing the issue

Reply:
If there is a better way, I'm all ears! 

From what I can tell cryo setting the mob to fall unconscious means it gets the TRAIT_FLOORED trait from set_stat. This then calls on_floored_trait_gain, which calls on_floored_start. This proc then makes the sprite rotate.

Reply:
This is less than ideal.

Make cryo add and remove a more generic trait besides this one, something to signify the contrary of `FLOORED`. Maybe `FORCED_STANDING` or the likes. Ideally we should port Naksu's traits in Goon for this, but a normal trait should do it in the meantime.

Then you substitute this check for that one there. Do it before the mobility flag change, though, same as the buckled check.

Reply:
Alright, will take a look!

Reply:
@Rohesie Changes made, this closer to what should be done?